kostal/README.md

```diff
@@ -1,14 +1,21 @@
 # KOSTAL
 
+VERSION = "1.3.1"
+
 # Requirements
 
-This plugin is designed to retrieve data from a [KOSTAL](http://www.kostal-solar-electric.com/) inverter module (e.g. PICO inverters).
+This plugin is designed to retrieve data from a [KOSTAL](http://www.kostal-solar-electric.com/) inverter module (e.g. PIKO inverters)
+Since UI-version 6 json-format is supported
+
+Tested with Kostal Piko 3.0 (single phase)  UI-Version 06.20
+dc-line 2, ac line 2 and ac line 3 values unchecked/untested
 
 ## Supported Hardware
 
-Is currently working with the following KOSTA inverter modules:
+Is currently working with the following KOSTAL inverter modules:
 
-  * KOSTAL PIKO 7.0
+  * KOSTAL PIKO 3.0
+  (should work with all KOSTAL PIKO inverters)
```

Comment:
Maybe we can change this to: Supported converters:
* PIKO 7.0 (old ones)
* PIKO 3.0 (new ones, next generation)

Additionally we can link to the home pages:
old ones http://kostal-solar-electric.com/de-DE/Produkte_Service/PIKO-Wechselrichter_bewaehrte_Generation,
new ones http://kostal-solar-electric.com/de-DE/Produkte_Service/PIKO-Wechselrichter_neue_Generation